### DISCUSSION TITLE
host vs target

### DISCUSSION THREAD

**Discussion by aoplik** [namhyung/uftrace#1710] [General]
This is mostly a naive question. 
I follow installation steps and can run uftrace locally at my host. 
My target machine though isn't straightforward. 
To check quickly I copied uftrace binary and the dependent libs to target system. 
However getting this error 

 ./uftrace fib
ERROR: ld.so: object 'libmcount-fast.so' from LD_PRELOAD cannot be preloaded: ignored.
uftrace: record.c:1437:update_session_maps
 ERROR: cannot find map files: No such file or directory

Is there a recommended steps to get uftrace compiled and installed in one place and used elsewhere ?

Thanks!

**Comment by alexmyczko** [comment]
Can you tell uname -a of both machines?
It is best to build it on the target machine when no binaries are available...

**Comment by namhyung** [comment]
Sorry for the late reply.  `uftrace` will search the library in the specified location in your config as well as the default library location (e.g. `/usr/lib`).  If you built it from source, please check the `libdir` in the `.config` file.  And place all libmcount libraries at the same location in your target system.
